src/SDL12_compat.c

```diff
@@ -5581,6 +5581,24 @@ UpdateRect12to20(SDL12_Surface *surface12, const SDL12_Rect *rect12, SDL_Rect *r
     }
 }
 
+/* For manual throttling of screen updates. */
+static int
+GetDesiredMillisecondsPerFrame()
+{
+    SDL_DisplayMode mode;
+    if (VideoSurface12->flags & SDL12_FULLSCREEN) {
+       if (!SDL20_GetWindowDisplayMode(VideoWindow20, &mode)) {
```

Comment:
For SDL functions that return 0 or an error, you should check == 0, not !func.

Reply:
Done.